src/CMakeLists.txt

```diff
@@ -113,7 +113,11 @@ if (${AMBER_ENABLE_SHADERC})
 endif()
 
 if (NOT MSVC AND NOT ANDROID)
-  target_link_libraries(libamber pthread)
+  if ("${CMAKE_SYSTEM_NAME}" STREQUAL "QNX")
+    target_link_libraries(libamber)
```

Comment:
This is a no-op. I'd recommend just adding the `QNX` to the `if (NOT MSVC AND NOT ANDROID)` condition.

Reply:
Done